packages/modular-scripts/src/lint.ts

```diff
@@ -50,7 +50,7 @@ async function lint(
     displayName: 'lint',
     rootDir: modularRoot,
     testMatch: targetedFiles,
-    testPathIgnorePatterns: ['/node_modules/', '/dist/'],
+    testPathIgnorePatterns: ['/node_modules/', '/dist/', '/config/'],
```

Comment:
Let's leave this for now and fix in a different commit by changing the file structure?

Reply:
If I remove it in the next commit I think it will work - wdym by "changing the file structure"?

Reply:
ah, ok, you want to move it under src, ok

Reply:
👍🏻 Removed, we'll capture this in another ticket